Wordle.Api/Wordle.Api/Services/WordEditService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Wordle.Api.Dtos;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Services;
+
+public class WordEditService(WordleDbContext Db)
+{
+    public WordleDbContext Db { get; set; } = Db;
+
+    public async Task AddWord(WordDto wordToAdd)
```

Comment:
This is where you could check to see if the word is in the db :) (for the no duplicate words requirement)

Reply:
we did it on the front end but thanks for the suggestion

Reply:
We should definitely add a secondary check here too though - thanks!